docs/java/guides/dependencies.md

```diff
@@ -89,3 +89,32 @@ Google the error message.
 Update the SDK version frequently.
 - This mitigates the risk per update and ensures you are up to date.
 
+### Overriding Dependency Versions of the SDK BOM
+
+Sometimes you may want to override the version of a specific dependency the SDK is using.
+You can achieve this by listing it in the dependency management _before the SDK BOM_.
+
+For example to override the version of SLF4J:
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>your-sfl4j-version</version>
```

Comment:
```suggestion
            <version>your-slf4j-version</version>
```

or even a property...

```suggestion
            <version>${slf4j.version}</version>
```

Reply:
I went with the `your-version` style because we use it in other places as well :)